03-Export-Tags-Authors-Stories.py

```diff
@@ -8,14 +8,18 @@
 
 
 def write_csv(filename, columns):
+  def write_byte(b):
+    s = str(b)
+    return s[2:len(s)-1]
+  
   html_parser = HTMLParser()
   with open(filename, 'w') as fp:
     myFile = csv.writer(fp)
     myFile.writerow(columns)
     for row in results:
       r = []
       for s in row:
-        r.append('' if s is None else html.unescape(str(s)).encode('utf-8'))
+        r.append('' if s is None else write_byte(html.unescape(str(s)).encode('utf-8')))
```

Comment:
It's possible that now we're using Python 3, where all strings are UTF-8 by default, we might be able to get rid of all this fancy footwork and just stick `s` as is in the `r` array (great variable names, past!me).

(I don't suppose you fancy writing a unit test for this method that we could stuff some UTF-8 characters into to test that theory?)